pkg/handler-launcher-com/notify/info/info.pb.go

```diff
@@ -13,15 +13,12 @@ It has these top-level messages:
 */
 package info
 
-import (
-	fmt "fmt"
-
-	proto "github.com/golang/protobuf/proto"
-
-	math "math"
+import proto "github.com/golang/protobuf/proto"
```

Comment:
there are some files changed which are not related to the PR, 
(all the ones with the import updating)
also they update it in a weird way, maybe you have some tool locally that cause it ?

Reply:
perhaps generated with something other than `make generate`

Reply:
try running `make && make generate` and committing the results to see if that resolves the issue.

Reply:
done